docs/application/dotnet/samples/uix/overview.md

```diff
@@ -44,7 +44,15 @@ for TD:
 <table>
 	<tbody>
 		<tr>
-			<td>There is no sample. </td>
+			<tr>
+			<td>
+			<p><img alt="" height="180" src="media/w67textreader.png" width="180"/></p>
+			</td>
+			<td>
+			<p><strong>(W) TextReader</strong> [In progress]</p>
+			<p>This sample application demonstrates how to transform input text into sound data using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Uix.Tts.html">Tizen.Uix.Tts</a>.</p>
```

Comment:
@surbhig12 This TextReader for wearable is no included in this page. As we have TextReader for Mobile and TV already in this page, how about using the same sentence for all these 3 TextReader apps?

Reply:
Hello @mijin85cho,

Please correct me if I am wrong, I saw all the sentences again and the format seems the same:
This sample application demonstrates how to.......... using......
I am not sure if I got your suggestion right.

Reply:
Hello, @surbhig12 

For now, the descriptions are
 - **TextReader for Mobile**
   This sample application demonstrates how to synthesize voice from text using Tizen.Uix.Tts.
 - **TextReader for Wearable**
   This sample application demonstrates how to transform input text into sound data using Tizen.Uix.Tts.
 - **TextReader for TV**
   This sample application demonstrates how to synthesize voice from text using Tizen.Uix.Tts.

TextReader for Wearable and TextReader for Mobile & TV are using different sentences.
I'm just suggesting to use a same description to these TextReader, if they are describing same function for Mobile, Wearable and TV.

Reply:
@mijin85cho,
Thank you for clarifying. I agree with your point. We can use the same description.

Reply:
Fixed to use same description...